part-1/part-1.md

```diff
@@ -48,28 +49,28 @@ _c) Do displayed data recalculations on each periodical async data update_
 Ok. SetTimeout is good here, but it does not provide any complicated failure logic (repeat on failure). It is possible to implement it - but with some code overhead.
 
 _d) Cancel async operations in case it is not needed anymore_
-You can cancel setTimeout/setInterval - but you cannot cancel fetch (yes, you can with AbortController but check browser support first (IE11))
+You can cancel setTimeout/setInterval - but you cannot cancel fetch (well, you can with AbortController, but check browser support first (IE11))
 
 But again - _ **combination** _ of async requests result or _ **repeat/retry** _ on error logic implementation remains very challenging.
 
 Do we have a solution?
-YES
+YES!
 
 Observables from RxJS library.
 
-## What is Rx.JS
+## What is RxJS
 
-RxJS is a library for reactive programming using Observables, to make it easier to compose asynchronous or callback-based code. RxJS is the JavaScript implementation of Reactive programming (or the Reactive Extentions - Rx) paradigm.
+RxJS is a library for reactive programming using `Observables` which make it easier to compose asynchronous or callback-based code. RxJS is the JavaScript implementation of Reactive programming (or the Reactive Extentions - Rx) paradigm.
```

Comment:
nitpick: "...callback-based operations" instead of "...calback-based code"?

Reply:
Agreed